### PR TITLE
open-scene-graph 3.4.0

### PR DESCRIPTION
**Pull request by fwiesel**
Updates OpenSceneGraph to the [latest stable release](http://www.openscenegraph.org/index.php/download-section/stable-releases)

### CODE REVIEW

File: Library/Formula/open-scene-graph.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class OpenSceneGraph < Formula
   desc "3D graphics toolkit"
   homepage "http://www.openscenegraph.org/projects/osg"
-  url "http://trac.openscenegraph.org/downloads/developer_releases/OpenSceneGraph-3.3.3.zip"
-  sha256 "b81cda123ffb3bd108e0fe4be4fff1351d6636e6fb0a1475b2c4fb9618d3ae2b"
+  url "http://trac.openscenegraph.org/downloads/developer_releases/OpenSceneGraph-3.4.0.zip"
+  sha256 "5c727d84755da276adf8c4a4a3a8ba9c9570fc4b4969f06f1d2e9f89b1e3040e"
   revision 1
```
Comment (DomT4) [inline Library/Formula/open-scene-graph.rb]: Please cut this line on version updates, Thanks!

**Comment by fwiesel** [comment]
Hi, I removed line 6, presuming you meant that one. Cheers

**Comment by DomT4** [comment]
Yup! :+1:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/fe44c894d30aa0b2c2c5e28a9e8098270ddc9e22. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**